front/src/modules/ui/modal/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReactNode, useState } from 'react';
+import styled from '@emotion/styled';
+import { AnimatePresence, LayoutGroup } from 'framer-motion';
+import debounce from 'lodash.debounce';
+
+import { Button, ButtonVariant } from '@/ui/button/components/Button';
+import { TextInput } from '@/ui/input/text/components/TextInput';
+import { Modal } from '@/ui/modal/components/Modal';
+import { Section, SectionAlignment } from '@/ui/section/components/Section';
+import { H1Title, H1TitleFontColor } from '@/ui/typography/components/H1Title';
+
+interface ConfirmationModalProps {
+  isOpen: boolean;
+  title: string;
+  subtitle: ReactNode;
+  setIsOpen: (val: boolean) => void;
+  handleConfirmDelete: () => void;
```

Comment:
Confirmation Modal naming should not mention "delete"
Also, we like to name the callback: onXXX

so I would name this props: onConfirmClick